pandas/tests/test_panel.py

```diff
@@ -1560,6 +1560,17 @@ def test_truncate_fillna_bug(self):
         # it works!
         result.fillna(value=0.0)
 
+    def test_truncate_outofbounds(self):
```

Comment:
you need to structure the test so its runnable in all of the sub-classses Testers so its tested on ``Series/DataFrame`` etc.